operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -255,6 +258,15 @@ type ClusterConfigValue struct {
 	UseRawValue bool `json:"useRawValue,omitempty"`
 }
 
+// ExternalSecretKeySelector selects a key of an external Secret.
+// +structType=atomic
+type ExternalSecretKeySelector struct {
+	Name string `json:"name" protobuf:"bytes,2,opt,name=name"`
```

Comment:
If the protobuf annotations where just copied from the `corev1` package, can we drop them? I don't think they affect anything we do and the inconsistency will be very confusing in hind sight, especially as this type is going into CEL.

Or are these required because it is indeed going into CEL?

Reply:
I think those were C&Ped - I just pulled these from @alenkacz 's pr. I'll fix that up.